iothub/service/src/DigitalTwin/Authentication/PnpConnectionStringCredential.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.Devices.Authentication
     /// <summary>
     /// Allows authentication to the API using a Shared Access Key
     /// </summary>
-    internal class SharedAccessKeyCredentials : IotServiceClientCredentials
+    internal class PnpConnectionStringCredential : IotServiceClientCredentials
```

Comment:
Given we used DigitalTwin instead of PnP for the client, shouldn't we stick with that prefix? I think it can be confusing for someone else understanding that DT and PnP are the same thing.

Reply:
I am always confused about this. DigitalTwin of Pnp. It makes it confusing. But I see your point.

Reply:
Given we have gone with DigitalTwin anyway for the client, i'll updated these to DigitalTwin prefix.